index.html

```diff
@@ -72,103 +78,118 @@ <h2 class="intro2">I’am a Software Developer</h2>
       <main class="main-container">
         <div class="main-part">
           <h1>My Recent Work</h1>
-          <img
-            src="./img/Vector.png"
-            class="sepration-line"
-            alt="seprating-line"
-          />
+          <hr class="seprator">
+          </div>
+          <div class="recent-work">
+          <div class="placeholder-multistories">
           <img src="./img/placeholder.png" class="placeholder" alt="image1" />
+          </div>
+          <div class="multistories">
           <h2>Multi-Post Stories</h2>
           <p>
             A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
+            sign-ups required. This has been the industry's standard dummy text ever
             since the 1500s, when an unknown printer took a standard dummy text.
           </p>
```

Comment:
- [ ] The main project style is great but the font-family used does not match the Figma template. Kindly use the font 'Inter' as you used on other parts of this project to style the paragraph and title of this project to match the template.